lib/ansible/config/base.yml

```
@@ -1492,13 +1492,14 @@ TRANSFORM_INVALID_GROUP_CHARS:
   description:
     - Make ansible transform invalid characters in group names supplied by inventory sources.
     - If 'never' it will allow for the group name but warn about the issue.
+    - When 'ignore', it does the same as 'never' sans the warning.
```

It's a commonly-understood French word, but would still trip up translations.
```suggestion
    - When 'ignore', it does the same as 'never', without issuing a warning.
```

---

I guess it should also be changed two lines below then.